connector/mqtt/vernemq/k2v-bridge/bin/entryPoint.sh

```diff
@@ -9,28 +9,71 @@ if [ ! -z "${DEBUG+x}" ]; then
     set -x
 fi
 
+# Set environment variables that are used by the program
+export K2V_APP_BASEDIR=${K2V_APP_BASEDIR:-"/opt/k2v_bridge"}
+export K2V_APP_CONNECTION_RETRY_COUNT=${K2V_APP_CONNECTION_RETRY_COUNT:-"3"}
+export K2V_APP_CONNECTION_RETRY_TIMEOUT=${K2V_APP_CONNECTION_RETRY_TIMEOUT:-"3"}
+export K2V_APP_EJBCA_ADDRESS=${K2V_APP_EJBCA_ADDRESS:-"localhost:5583"}
+export K2V_APP_HOSTNAME=${K2V_APP_HOSTNAME:-$HOSTNAME}
+export K2V_APP_HOSTNAME=${K2V_APP_HOSTNAME:-"k2v-bridge"}
+
+export K2V_LOGGER_TRANSPORTS_CONSOLE_LEVEL=${K2V_LOGGER_TRANSPORTS_CONSOLE_LEVEL:-"info"}
+export K2V_LOGGER_VERBOSE=${K2V_LOGGER_VERBOSE:-"false"}
+
+export K2V_MESSENGER_CONSUME_TOPIC_SUFFIX=${K2V_MESSENGER_CONSUME_TOPIC_SUFFIX:-"dojot.device-manager.device"}
+
+export K2V_MQTT_CLIENT_ID=${K2V_MQTT_CLIENT_ID:-${K2V_APP_HOSTNAME}}
+export K2V_MQTT_CLIENT_KEEPALIVE=${K2V_MQTT_CLIENT_KEEPALIVE:-"60"}
+export K2V_MQTT_CLIENT_SECURE=${K2V_MQTT_CLIENT_SECURE:-"true"}
+export K2V_MQTT_CLIENT_PUBLISH_QOS=${K2V_MQTT_CLIENT_PUBLISH_QOS:-"1"}
+export K2V_MQTT_CLIENT_PUBLISH_TOPIC_SUFFIX=${K2V_MQTT_CLIENT_PUBLISH_TOPIC_SUFFIX:-"/config"}
+export K2V_MQTT_CLIENT_USERNAME=${K2V_MQTT_CLIENT_USERNAME:-${K2V_APP_HOSTNAME}}
+
+export K2V_MQTT_SERVER_ADDRESS=${K2V_MQTT_SERVER_ADDRESS:-"vernemq-k8s"}
+export K2V_MQTT_SERVER_PORT=${K2V_MQTT_SERVER_PORT:-"8883"}
+
+export K2V_MQTT_TLS_CA_FILE=${K2V_MQTT_TLS_CA_FILE:-"${K2V_APP_BASEDIR}/app/cert/ca.crt"}
+export K2V_MQTT_TLS_CERTIFICATE_FILE=${K2V_MQTT_TLS_CERTIFICATE_FILE:-"${K2V_APP_BASEDIR}/app/cert/${K2V_APP_HOSTNAME}.crt"}
+export K2V_MQTT_TLS_KEY_FILE=${K2V_MQTT_TLS_KEY_FILE:-"${K2V_APP_BASEDIR}/app/cert/${K2V_APP_HOSTNAME}.key"}
+
+export K2V_KAFKA_ACKS=${K2V_KAFKA_ACKS:-"-1"}
```

Comment:
Some of these properties doesn't apply to the Kafka Consumer. See the comments in the README.md